framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/settings/builder/KieServerS2ISettingsBuilder.java

```diff
@@ -119,11 +119,22 @@
      *
      * @param gitRepoUrl Repository url.
      * @param gitReference Repository reference (branch/tag). E.g. 'master'.
-     * @param gitContextDir Repositoy context (location of pom file).
-     * @return Buillder
+     * @param gitContextDir Repository context (location of pom file).
+     * @return Builder
      */
     KieServerS2ISettingsBuilder withSourceLocation(String gitRepoUrl, String gitReference, String gitContextDir);
 
+    /**
+     * Return configured builder with Source location
+     *
+     * @param gitRepoUrl Repository url.
+     * @param gitReference Repository reference (branch/tag). E.g. 'master'.
+     * @param gitContextDir Repository context (location of pom file).
+     * @param artifactDirs Directories containing built kjars.
```

Comment:
ARTIFACT_DIR accept list of artifacts, please add to doc in which format are artifacts accepted. (dir1,dir2,dir3)